src/boot/idle.c

```diff
@@ -1,9 +1,7 @@
 #include "global.h"
 #include "terminal.h"
 
-// clang-format off
 #pragma increment_block_number "gc-eu:64 gc-eu-mq:64 gc-jp:64 gc-jp-ce:64 gc-jp-mq:64 gc-us:64 gc-us-mq:64"
```

Comment:
should all the versions be in separate string literals here? it looks a bit awkward when fix_bss.py adds a new one to the front in a separate string but the rest are still together

Reply:
I'm leaving this formatting handling to be done manually https://github.com/cadmic/oot/pull/6#discussion_r1709854713